browser/model/installable-item.js

```diff
@@ -72,6 +72,10 @@ class InstallableItem {
     return this.productVersion;
   }
 
+  getProductSize() {
```

Comment:
I have added the function as to calculate total download size which we will show. Do you want to get this code in that PR. I will make that change